### ISSUE TITLE
when returning to the previous page in google nvda does not return to the same point

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8809]
### Steps to reproduce:
enter the google search page using chrome.
Do a search.
enter the 4th result for example.
Press a down arrow to read a few lines on the open page.
Press alt + left arrow to return to google page.
### Actual behavior:
The focus of the nvda returns to the top of the page or at least it does not return to the same point where it was when it flies from the page, this wants to say about the 4th result.
This fails in chrome 69 however it runs in firefox.
### Expected behavior:
The focus of the nvda should be where it was when it left the google page this wants to disser in the 4th result.
#### NVDA Installed/portable/running from source:
instaled
#### NVDA version:
alfa 16134
#### Windows version:
10 17134.320
#### Name and version of other software in use when reproducing the issue:
chrome 69
### Other questions:
In the attached log there are some errors
#### Does the issue still occur after restarting your PC?
yes
#### Have you tried any other versions of NVDA?
yes
[google.txt](https://github.com/nvaccess/nvda/files/2432258/google.txt)

**Comment by ehollig** [comment]
This works correctly for me using alpha-16157 and latest stable Chrome. @fernando-jose-silva does this happen every time you perform a search using Chrome?

**Comment by Adriani90** [comment]
@fernando-jose-silva I cannot reproduce this issue with NVDA 2019.3 Beta 1 on Chrome 79. Do you still habe this problem? Do you have enabled by chance automatically  read page after page load in NVDA's browse mode settings? If yes, please disable it and see if the issue still occurs.

**Comment by fernando-jose-silva** [comment]
Thanks, I tested on the last nvda alpha and chrome and everything works correctly.

**Comment by Adriani90** [comment]
Closing as works for me.
